.github/workflows/build.yml

```
@@ -463,6 +463,69 @@ jobs:
           name: 'PackSquash CLI AppImage zsync (arm64)'
           path: PackSquash-*-aarch64.AppImage.zsync
 
+  docker-image:
+
+    runs-on: ubuntu-latest
+
+    env:
+      # Use docker.io for Docker Hub if empty
+      REGISTRY: ghcr.io
+      # github.repository as <account>/<repo>
+      IMAGE_NAME: ${{ github.repository }}
+
+    needs: [ linux-x64-gnu, linux-aarch64-gnu ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Docker image metadata
+        id: dockermeta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # generate Docker tags: edge for every commit on master, semver for every semantic tag
+          tags: |
+            type=edge,branch=master
+            type=semver,pattern={{version}}
+        
+      - name: Download PackSquash X86-64 AppImage
+        uses: actions/download-artifact@v3.0.0
```

```suggestion
      - name: Download PackSquash X86-64 AppImage
        uses: actions/download-artifact@v3
```
v3 is better than v3.0.0.

---

Resolved in https://github.com/ComunidadAylas/PackSquash/pull/111/commits/0d7bc6680135eb3c23e7628235625b29bfd52d5b